src/com/musala/xml/parser/Main.java

```diff
@@ -0,0 +1,181 @@
+package com.musala.xml.parser;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import javax.xml.bind.JAXBContext;
+import javax.xml.bind.JAXBException;
+import javax.xml.bind.Unmarshaller;
+
+import com.musala.xml.model.School;
+import com.musala.xml.model.SchoolClass;
+import com.musala.xml.model.Student;
+import com.musala.xml.model.Teacher;
+
+public class Main {
+
+    public static void main(String[] args) throws IOException, JAXBException {
+        String pathname = "C:\\Users\\stefan.simeonov\\Desktop\\ThePath.xml";
+        Path pathname2 = Paths.get("C:\\Users\\stefan.simeonov\\Desktop\\ThePath.xml");
+        // File file = new File(pathname);
+        /*
+         * Scanner in = null;
+         * try {
+         * in = new Scanner(file);
+         * 
+         * while (in.hasNextLine()) {
+         * System.out.println(in.nextLine());
+         * }
+         * } catch (FileNotFoundException e) {
+         * System.out.println(e.getMessage());
+         * } finally {
+         * if (in != null)
+         * in.close();
+         * }
+         */
+
+        /*
+         * FileInputStream input = null;
+         * try {
+         * input = new FileInputStream(file);
+         * int content;
+         * while((content=input.read())!=-1){
+         * System.out.print((char)content);
+         * }
+         * } catch ( IOException ex) {
+         * System.out.println(ex.getMessage());
+         * }finally{
+         * if(input!=null) input.close();
+         * }
+         */
+        // FileReader inpt=new FileReader(file);
+        // char[] mass=new char[1000];
+        // inpt.read(mass);
+        // for(char symbol:mass){
+        // System.out.print(symbol);
+        // }
+        /*
+         * int content;
+         * while((content=inpt.read())!=-1){
+         * System.out.print((char)content);
+         * }
+         */
+        // byte[] arr = Files.readAllBytes(pathname2);
+        // System.out.println(new String(arr, "UTF-8"));
+        String pathname3 = "ThePath.xml";
+        getFile(Constants.filepath);
+
+        ClassLoader classload = Main.class.getClassLoader();
+        File configFile = new File(classload.getResource(pathname3).getFile());
+        JAXBContext jaxb = JAXBContext.newInstance(School.class);
+        Unmarshaller jaxbUnmarsheller = jaxb.createUnmarshaller();
+        School mySchool = (School) jaxbUnmarsheller.unmarshal(configFile);
+        System.out.println(mySchool.getStudents().getStudent().get(0).getAge());
+        System.out.println(mySchool.getStudents().getStudent().get(1).getAge());
+        System.out.println(mySchool.getSchoolClasses().get(0).getTeacherReference());
+        System.out.println(mySchool.getTeachers().getTeacher().get(0).getAge());
+        Print(mySchool);
```

Comment:
All the system.outs before the Print(mySchool) statement are for testing purposes. Please remove them. Also, remove the XML file printing - this was for testing purposes as well. Leave only meaningful output data ("The school has N students..." etc.)